jams.js

```diff
@@ -27,10 +29,12 @@ const _jams =ast=> {
             return _jams(ast.children[0])
         }
         case 'str': {
-            if (ast.text === '""') return ''
-            if (ast.children.length !== 1) throw new Error(`Invalid string`)
-            const utf8_encoded = String.raw`"${ast.children[0].text}"`
-            const json = JSON.parse(utf8_encoded)
+            return (ast.text.includes('""') && ast.children.length === 0) ? "" : _jams(ast.children[0])
+        }
+        case 'bare':
+        case 'quote': {
+            const quoted = String.raw`"${ast.text}"`
+            const json = JSON.parse(quoted)
```

Comment:
Did we verify json strings escape unicode and not utf16?

Reply:
> Did we verify json strings escape unicode...?

Yep. Longer answer is, I verified that a file of `\n` needs to transform to `String.raw`"\\n"\` (note the quotes) for JSON.parse to parse, otherwise it's a parsing exception. Did I answer your question?

Reply:
> and not utf16?

I think I am not understanding the question. Could you please break it down a bit? 
JSON strings ----can transform to---> JS Objects ---> UTF-8 encoded files. 

Could you please let me know if this and the comment in the diff `test.js` helps?

Reply:
I’m talking about escape sequences, how you specify them like `\uff` vs `U+FF` etc, json spec encodes utf16 I’m 99% sure and I want it to be unicode instead 
Will merge this but investigate some more later